prime-router/docker-compose.yml

```diff
@@ -109,6 +109,9 @@ services:
       - prime_dev
     command: "'nginx' '-g' 'daemon off;'"
     entrypoint: "/docker-entrypoint.sh"
+    build:
+      context: ../frontend
+      dockerfile: ../frontend/Dockerfile_dev
```

Comment:
Nitpick: Prefer calling that file `Dockerfile.dev` (separator being a `.`. instead of an `_`)